### ISSUE TITLE
iOS Podspec file missing homepage

### ISSUE THREAD

**Issue by imoby** [KjellConnelly/react-native-rate#9]
I get an error when running pod install for RNRate. It says that homepage property is not configured in the podspec file.

```
Pod::Spec.new do |s|
  s.name         = "RNRate"
  s.version      = "1.0.0"
  s.summary      = "RNRate"
  s.description  = <<-DESC
                  RNRate
                   DESC
  s.homepage     = ""
  s.license      = "MIT"
  # s.license      = { :type => "MIT", :file => "FILE_LICENSE" }
  s.author             = { "author" => "author@domain.cn" }
  s.platform     = :ios, "7.0"
  s.source       = { :git => "https://github.com/author/RNRate.git", :tag => "master" }
  s.source_files  = "RNRate/**/*.{h,m}"
  s.requires_arc = true


  s.dependency "React"
  #s.dependency "others"

end
```

![image](https://user-images.githubusercontent.com/6380855/40155757-0a488c32-595b-11e8-93c0-9f845ee36399.png)

**Comment by imoby** [comment]
@KjellConnelly I've created a pull request to fix this issue: https://github.com/KjellConnelly/react-native-rate/pull/10

**Comment by KjellConnelly** [comment]
Thanks imoby! Haven’t used cocoapods with React Native before, and this module was initialized via a script, hence me leaving in the podspec that I never looked out. Will merge it now :)

**Comment by imoby** [comment]
@KjellConnelly any plans on when this will be published to npm?

**Comment by KjellConnelly** [comment]
@imoby Just published 1.0.9 to npm with your change :)
